apps/docs/src/guide/creating-a-fuel-dapp/index.md

```diff
@@ -183,14 +183,22 @@ Second, we will add a new button to the UI that will call the `onDecrementPresse
 </Button>
 ```
 
-Congratulations! That's all. You should now be able to see the counter dApp running at `http://localhost:3000` with our newly added decrement functionality.
+Congratulations! You should now be able to see the counter dApp running at `http://localhost:3000` with our newly added decrement functionality.
 
 You can find the complete source code of the dApp we built [here](https://github.com/FuelLabs/fuels-ts/tree/master/apps/create-fuels-counter-guide).
 
 ![End result of this guide](../../public/creating-a-fuel-dapp-create-fuels-end-result.png)
 
 Whenever you want to add a new feature to your dApp and quickly prototype things, you can follow the same steps we followed in this guide.
 
+### 3. Extending the Test Suite (Optional)
```

Comment:
Why is this numbered?

```suggestion
## Extending the Test Suite (Optional)
```

Reply:
IMO it flows on from the other decrement functionality steps https://docs.fuel.network/docs/fuels-ts/creating-a-fuel-dapp/#adding-decrement-functionality